### PR TITLE
User profiles

### LINKED ISSUE #70: Implement profile header image

### LINKED ISSUE #73: User profile model

### LINKED ISSUE #74: User profile routes

### PR DESCRIPTION
**Pull request by KevinGomes**
Profile routes and models for closing issues for uploading profile images, and changing user and password.

### CODE REVIEW

File: Flora-Fauna-Backend/models/profile.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      WHERE id = $2
+      RETURNING profile_image
+      `,
+      [resizedImage, user_id],
+    );
+
+    //Deletes file from uploads file, as to not become bloated
+    fs.unlink(`${image.path}`, () => {
+      //Need to improve before going into production || Alternative: Put success info into txt
+      console.log("success");
```
Comment (Rebecca-CYLi) [inline Flora-Fauna-Backend/models/profile.js]: found a console log

File: Flora-Fauna-Backend/models/profile.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+     WHERE id = $2
+     RETURNING header_image
+     `,
+     [resizedImage, user.id],
+   );
+
+   //Deletes file from uploads file, as to not become bloated
+   fs.unlink(`${image.path}`, () => {
+     //Need to improve before going into production || Alternative: Put success info into txt
+     console.log("success");
```
Comment (Rebecca-CYLi) [inline Flora-Fauna-Backend/models/profile.js]: found a console log

File: Flora-Fauna-Backend/models/profile.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        WHERE id = $2
+        RETURNING email
+        `,
+        [cleanEmail, user_id],
+      );
+      return results.rows[0];
+    }
+    
+    static async updatePassword(password, user_id) {
+      console.log(password, user_id)
```
Comment (Rebecca-CYLi) [inline Flora-Fauna-Backend/models/profile.js]: found a console log

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
found console logs, please fix them

File: Flora-Fauna-Backend/models/profile.js (hunk 1 above)
Comment (KevinGomes) [reply Flora-Fauna-Backend/models/profile.js]: fixed

**Review by Rebecca-CYLi — APPROVED** [review]
revisions done, G2G

**Pull request merged**